compiler/locomotiv/src/Node/TensorBroadcast.cpp

```diff
@@ -42,6 +42,10 @@ using namespace locomotiv;
 void execute_node(loco::TensorBroadcast *tensor_broadcast)
 {
   auto input_data = annot_data(tensor_broadcast->input());
+  if (input_data == nullptr)
+  {
+    throw std::runtime_error("Any annotation is found in TensorBroadcast");
```

Comment:
Q) what is `Any` ?

Reply:
As I thought that `annot_data` returns annotated data from node, `input_data == nullptr` means that there are no annotations..!
Could you recommend other error msg? :)